sdk/communication/azure-communication-common/src/main/java/com/azure/communication/common/CommunicationIdentifier.java

```diff
@@ -9,6 +9,81 @@
  */
 public abstract class CommunicationIdentifier {
 
+    /**
+     * Prefix for a phone number.
+     */
+    protected static final String PHONE_NUMBER = "4:";
+
+    /**
+     * Prefix for a bot.
+     */
+    protected static final String BOT_GLOBAL = "28:";
+
+    /**
+     * Prefix for a bot with public cloud.
+     */
+    protected static final String BOT_PUBLIC_CLOUD = "28:orgid:";
+
+    /**
+     * Prefix for a bot with DOD cloud.
+     */
+    protected static final String BOT_DOD_CLOUD = "28:dod:";
+
+    /**
+     * Prefix for a global bot with DOD cloud.
+     */
+    protected static final String BOT_DOD_CLOUD_GLOBAL = "28:dod-global:";
+
+    /**
+     * Prefix for a bot with GCCH cloud.
+     */
+    protected static final String BOT_GCCH_CLOUD = "28:gcch:";
+
+    /**
+     * Prefix for a global bot with GCCH cloud.
+     */
+    protected static final String BOT_GCCH_CLOUD_GLOBAL = "28:gcch-global:";
+
+    /**
+     * Prefix for an anonymous Teams user.
+     */
+    protected static final String TEAM_USER_ANONYMOUS = "8:teamsvisitor:";
+
+    /**
+     * Prefix for a Teams user with public cloud.
+     */
+    protected static final String TEAM_USER_PUBLIC_CLOUD = "8:orgid:";
+
+    /**
+     * Prefix for a Teams user with DOD cloud.
+     */
+    protected static final String TEAM_USER_DOD_CLOUD = "8:dod:";
+
+    /**
+     * Prefix for a Teams user with GCCH cloud.
+     */
+    protected static final String TEAM_USER_GCCH_CLOUD = "8:gcch:";
```

Comment:
same as https://github.com/Azure/azure-sdk-for-java/pull/33634/files#r1120056446